Core/MIPS/JitCommon/JitState.h

```diff
@@ -198,6 +198,8 @@ namespace MIPSComp {
 		POINTERIFY = 0x00400000,
 		STATIC_ALLOC = 0x00800000,
 		CACHE_POINTERS = 0x01000000,
+
+		ALL_FLAGS = 0x01FFFFFF,
```

Comment:
I did leave a gap, so currently 0x01F00FDF.  Not that it matters, just noticed you used this instead of 0xFFFFFFF so mentioning.

-[Unknown]

Reply:
Oops, but yeah, doesn't really matter much.